__tests__/components/forecast-summaries.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Enzyme from 'enzyme';
```

Comment:
You can make your like a little easier by using a name import here. `import { shallow } from 'enzyme'` would mean you could just use `shallow()` in your tests. This has the added bonus of only importing what we need.